modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -1040,6 +1046,19 @@ protected int processStartTag() throws Exception {
 
 						iconTag.doTag(pageContext);
 
+						if (FFManagementToolbarConfigurationUtil.
```

Comment:
Seeing that this is how we deal with feature flags, I don't want to imagine how complex this could get - it's a good example though (note that I'm just leaving a comment here to be able to find it later when we talk about this)